pkg/kubectl/cmd/resource/get.go

```diff
@@ -51,6 +52,8 @@ import (
 
 // GetOptions contains the input to the get command.
 type GetOptions struct {
```

Comment:
I don't expect to see `Get` using a normal printer. Since `kubectl get` has so many special printers, I'd say that combining it is an anti-goal